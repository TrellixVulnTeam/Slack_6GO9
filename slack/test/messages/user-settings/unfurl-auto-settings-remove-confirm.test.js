const { UnfurlAutoSettingsRemoveConfirm } = require('../../../lib/messages/user-settings');

describe('UnfurlAutoSettingsRemoveConfirm rendering', () => {
  test('works', async () => {
    expect(new UnfurlAutoSettingsRemoveConfirm('integrations/slack').getAttachment()).toMatchSnapshot();
  });
});
