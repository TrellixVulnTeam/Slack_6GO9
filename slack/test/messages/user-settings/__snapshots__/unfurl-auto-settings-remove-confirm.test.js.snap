// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnfurlAutoSettingsRemoveConfirm rendering works 1`] = `
Object {
  "color": "#24292f",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": ":white_check_mark: Links you post to \`integrations/slack\` will no longer receive automatic rich previews.",
}
`;
